src/img/icons/calendar.svg

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
I'll talk with Ilona and we will put all SVG/icons into font, so it will be lighter and easier to manage.